closes #8 updating release documentation to reflect new release process

majormoses opened the pull request:
Based of the instructions here: https://github.com/sensu-plugins/sensu-plugins-feature-requests/issues/17#issuecomment-298465993

I went through this to make sure this is valid

majormoses committed:
closes #8 updating release documentation to reflect new release process

eheydrick commented:
Content is 👍 but turns out this page moved to a different place on the site. http://sensu-plugins.io/docs/b_and_r.html is backed by https://github.com/sensu-plugins/documentation/blob/master/docs/b_and_r.md. Can you open a PR on that one. We should delete the pages that aren't used anymore.

majormoses committed:
updating build and release documentation:
- as per https://github.com/sensu-plugins/documentation/pull/9#issuecomment-298695602 the documentations home has moved and updated the new location removing old to avoid confusion

majormoses commented:
@eheydrick have updated with the new document location.

eheydrick commented:
Changes are on http://sensu-plugins.io/docs/b_and_r.html now.
